executor/insert.go

```diff
@@ -205,6 +206,7 @@ func (e *InsertExec) Open(ctx context.Context) error {
 func (e *InsertExec) updateDupRow(row toBeCheckedRow, handle int64, onDuplicate []*expression.Assignment) error {
 	oldRow, err := e.getOldRow(e.ctx, e.Table, handle)
 	if err != nil {
+		log.Errorf("[insert on dup] can not find record which handle is %d for the to-be-inserted row %v", handle, row.row)
```

Comment:
Why not use %s?

Reply:
The lint tool does not allow %s.

Reply:
log.Errorf("[insert on dup] cannot find the record whose handle is %d for the to-be-inserted row %v", handle, row.row)